### ISSUE TITLE
Styling guideline

### ISSUE THREAD

**Issue by profOnno** [helge17/tuxguitar#278]
I didn't notice a styling guideline in `contribute.md`, so I assume there isn't one.
Working on the code I noticed the is not always a space after a comma. 

When using Eclipse, "auto" format can be activated on save, then all the style mismatches will be fixed. I found this [thread](https://stackoverflow.com/questions/3754405/best-eclipse-code-formatters) on stackoverflow. The is a choice to config the formatter using, google convention, sun coding conventions, or define you own.

Is this something to add to tuxguitar?

**Comment by guiv42** [comment]
You're right there's no styling guideline. And some files have serious style issues (e.g. mix of tabs & space indentation,...)
> Is this something to add to tuxguitar?

Don't hesitate to submit one PR if you're inspired ;)
In this case, it would be better if you can take actual code as a reference. Possibly styling guidelines could be defined in a dedicated .md doc? (contribute.md is already a bit long)

There's a lot of work to do in this area: coding rules, variables/classes names convention, etc. This fork started here about a year ago, and we are slowly gaining maturity ("contribute.md" file itself is very recent).

**Comment by guiv42** [comment]
I've tried a few different settings, unfortunately the code seems to be quite heterogeneous.
A possibility could be to use current Eclipse's default formatting. If no one opposes to that, I will add it in the documentation.

[tuxguitar_formatting.zip](https://github.com/user-attachments/files/18549743/tuxguitar_formatting.zip)

**Comment by helge17** [comment]
I don't have any big preferences, it's fine for me.

I would fix the following points as soon as tuxguitar-next is merged to master:

- Replace DOS line endings with Unix line endings
- Delete white spaces at line endings
- Add missing newlines at end of file
